src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -683,6 +684,9 @@ class LicenceService(
       statusCode = ACTIVE,
       dateLastUpdated = LocalDateTime.now(),
       updatedByUsername = username,
+      approvedByUsername = username,
+      approvedDate = LocalDateTime.now(),
+      approvedByName = body.fullName
     )
```

Comment:
Should the full name of the approver also go into the events below this? Looking at it, its assuming that the approver will also appear in the comDetail table - recorded at login, so it looks like the full name is already obtainable from there?

Reply:
Sorted this now, thanks